components/ui/card.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
+
+import { cn } from "@/lib/utils"
+
+const Card = React.forwardRef<
```

Comment:
Would you mind explaining why you need forwardRefs?

Reply:
This is from shadcn